test/stablecoin_converter.js

```diff
@@ -642,7 +651,37 @@ contract("StablecoinConverter", async (accounts) => {
         stablecoinConverter.submitSolution(batchIndex, owner, orderId, volume, prices, tokenIdForPrice)
       )
     })
-    it("reverts, if a trade touches a token, for which no price is provided", async () => {
+    it("reverts, if a trade touches a token, for which no price is provided in function findPriceIndex", async () => {
```

Comment:
How is this different from the one below"

```reverts, if a trade touches a token, for which no price is provided```

findPriceIndex is an internal implementation detail and shouldn't really be exposed to the unit test level since we should be able to e.g change its name or implementation without changing the unit test.

Reply:
I made the test descriptions better.
One time, we do not find the price in the binary search and the other time, we would have the same issue, but first, divide by 0 somewhere else.